src/Identity/ApiAuthorization.IdentityServer/src/Data/ApiAuthorizationDbContext.cs

```diff
@@ -45,6 +45,14 @@ public class ApiAuthorizationDbContext<TUser> : IdentityDbContext<TUser>, IPersi
         /// </summary>
         public DbSet<DeviceFlowCodes> DeviceFlowCodes { get; set; }
 
+        /// <summary>
+        /// Gets or sets the keys.
+        /// </summary>
+        /// <value>
+        /// The keys.
+        /// </value>
+        public DbSet<Key> Keys { get; set; }
```

Comment:
I see a couple of new `DbSet<T>` properties in this PR but haven't noticed new references to them. What are they for❔

Reply:
Yeah it comes from a new requirement from the updated version of IPersistedGrantDbContext, which now store Keys, I'm not sure exactly what they are used for, but its part of the contract for IdentityServer Stores